Sources/ExyteChat/ChatView/InputView/InputView.swift

```diff
@@ -64,60 +66,53 @@ public enum InputViewState {
 }
 
 public enum AvailableInputType {
```

Comment:
rather than now take in textAndAudio textAndMedia, the setInputTypes takes in an array of inputType, this is because you now have 4 different permutations of input type text, media, audio and giphy. Taking in an array is preferable to having textAndAudioAndGiphy textAndAudio etc